tests/integrationEnvironment.ts

```diff
@@ -2,11 +2,16 @@ import { Config } from "@jest/types"
 import BrowserEnvironment from "jest-environment-jsdom"
 import timers from "timers"
 
+interface ConfigTypes {
+  globalConfig: Config.GlobalConfig;
+  projectConfig: Config.ProjectConfig;
+}
 class IntegrationEnvironment extends BrowserEnvironment {
-  constructor(config: Config.ProjectConfig) {
-    super(
-      Object.assign({}, config, {
-        globals: Object.assign({}, config.globals, {
+  constructor({ globalConfig, projectConfig }: ConfigTypes, context: any) {
+    super({ ...globalConfig,
+      projectConfig: {
+        ...projectConfig,
+        globals: Object.assign({}, projectConfig.globals, {
```

Comment:
Jest v28 introduced a breaking change to the config object: https://jest-archive-august-2023.netlify.app/docs/28.x/upgrading-to-jest28/#test-environment